languages/go/config.json

```diff
@@ -177,13 +176,13 @@
         "slug": "slices",
         "uuid": "8bc0c6f6-1891-4688-9ed7-a63e3702f655",
         "concepts": ["slices"],
-        "prerequisites": ["arrays", "numbers"]
+        "prerequisites": ["basics", "iteration", "numbers"]
       },
       {
         "slug": "strings",
         "uuid": "b97a271c-4036-4ea8-8c6a-094715a48627",
         "concepts": ["strings"],
-        "prerequisites": ["basics"]
+        "prerequisites": ["slices"]
```

Comment:
The current `strings` exercise needs knowledge about slices to be able to be solved. For example getting a sub-text from a string is done via slice syntax: "Hello World!"[:5] return "Hello".

We could have a second strings exercise right after `basics` that works without knowledge about slices. Then we would have to rename the current `strings` exercise to something like `strings-advanced`.

What do you think @oanaOM?